### PR TITLE
🌐 Add Japanese translation for Advanced Tutorial - Response Directly

### PR DESCRIPTION
**Pull request by Attsun1031**
This PR translates the advanced/response-directly.md to Japanese.

### CODE REVIEW

File: docs/ja/docs/advanced/response-directly.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# レスポンスを直接返す
+
+**FastAPI** の *path operation* では、通常は任意のデータを返すことができます: 例えば、 `dict`、`list`、Pydanticモデル、データベースモデルなどです。
+
+デフォルトでは、**FastAPI** は [JSON Compatible Encoder](../tutorial/encoder.md){.internal-link target=_blank} で説明されている `jsonable_encoder` により、返される値を自動的にJSONに変換します。
```
Comment (tokusumi) [inline docs/ja/docs/advanced/response-directly.md:5]:
```suggestion
デフォルトでは、**FastAPI** は [JSON互換エンコーダ](../tutorial/encoder.md){.internal-link target=_blank} で説明されている `jsonable_encoder` により、返す値を自動的にJSONに変換します。
```
PRにあるタイトルを使用。

File: docs/ja/docs/advanced/response-directly.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+実際は、`Response` やそのサブクラスを返すことができます。
+
+!!! tip "豆知識"
+    `JSONResponse` それ自体は、 `Response` のサブクラスです。
+
+`Response` を返した場合は、**FastAPI** は直接それを返します。
+
+それは、Pydanticモデルのデータ変換や、コンテンツを任意の型に変換したりなどはしません。
+
+これは多くの柔軟性を提供します。任意のデータ型を返したり、任意のデータ宣言やバリデーションをオーバーライドしたりすることができます。
```
Comment (tokusumi) [inline docs/ja/docs/advanced/response-directly.md:24]:
```suggestion
これは多くの柔軟性を提供します。任意のデータ型を返したり、任意のデータ宣言やバリデーションをオーバーライドできます。
```
簡潔に

File: docs/ja/docs/advanced/response-directly.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## `jsonable_encoder` を `Response` の中で使う
+
+**FastAPI** はあなたが返す `Response` に対して何も変更を加えないので、コンテンツが準備できていることを保証しなければなりません。
+
+例えば、Pydanticモデルを `JSONResponse` に含めるには、すべてのデータ型 (`datetime` や `UUID` など) がJSON互換の型に変換された `dict` に変換しなければなりません。
+
+このようなケースでは、レスポンスにデータを含める前に `jsonable_encoder` を使ってデータ変換をすることができます。
```
Comment (tokusumi) [inline docs/ja/docs/advanced/response-directly.md:32]:
```suggestion
このようなケースでは、レスポンスにデータを含める前に `jsonable_encoder` を使ってデータを変換できます。
```
簡潔に

File: docs/ja/docs/advanced/response-directly.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+```Python hl_lines="6-7  21-22"
+{!../../../docs_src/response_directly/tutorial001.py!}
+```
+
+!!! note "技術詳細"
+    また、`from starlette.responses import JSONResponse` も利用することができます。
```
Comment (tokusumi) [inline docs/ja/docs/advanced/response-directly.md:39]:
```suggestion
    また、`from starlette.responses import JSONResponse` も利用できます。
```
簡潔に

File: docs/ja/docs/advanced/response-directly.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+    **FastAPI** は開発者の利便性のために `starlette.responses` を `fastapi.responses` と同じものとして提供しています。しかし、利用可能なレスポンスのほとんどはStarletteから直接提供されます。
```
Comment (tokusumi) [inline docs/ja/docs/advanced/response-directly.md:41]:
```suggestion
    **FastAPI** は開発者の利便性のために `fastapi.responses` という `starlette.responses` と同じものを提供しています。しかし、利用可能なレスポンスのほとんどはStarletteから直接提供されます。
```

File: docs/ja/docs/advanced/response-directly.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+## カスタム `Response` を返す
+
+上記の例では必要な部分を全て示していますが、あまり便利ではありません。`item` を直接返すことができるし、**FastAPI** はそれを `dict` に変換して `JSONResponse`　に含めてくれるなど。すべて、デフォルトの動作です。
+
+では、これを使ってカスタムレスポンスをどう返すか見てみましょう。
+
+<a href="https://en.wikipedia.org/wiki/XML" class="external-link" target="_blank">XML</a>レスポンスを返したいとしましょう。
+
+XMLを文字列にし、 `Response` に含め、それを返します。
```
Comment (tokusumi) [inline docs/ja/docs/advanced/response-directly.md:51]:
```suggestion
XMLを文字列にし、`Response` に含め、それを返します。
```

**Review by tokusumi — CHANGES_REQUESTED** [review]
Thank you for great work!

**Comment by Attsun1031** [comment]
@tokusumi Thank you for your review ! Every your suggestion are applied, so please check again.

**Review by tokusumi — APPROVED** [review]
Thank you for cooperating!

File: docs/ja/docs/advanced/response-directly.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+それは、Pydanticモデルのデータ変換や、コンテンツを任意の型に変換したりなどはしません。
+
+これは多くの柔軟性を提供します。任意のデータ型を返したり、任意のデータ宣言やバリデーションをオーバーライドできます。
+
+## `jsonable_encoder` を `Response` の中で使う
+
+**FastAPI** はあなたが返す `Response` に対して何も変更を加えないので、コンテンツが準備できていることを保証しなければなりません。
+
+例えば、Pydanticモデルを `JSONResponse` に含めるには、すべてのデータ型 (`datetime` や `UUID` など) がJSON互換の型に変換された `dict` に変換しなければなりません。
```
Comment (komtaki) [inline docs/ja/docs/advanced/response-directly.md:30]:
```suggestion
例えば、Pydanticモデルを `JSONResponse` に含めるには、すべてのデータ型 (`datetime` や `UUID` など) をJSON互換の型に変換された `dict` に変換しなければなりません。
```

File: docs/ja/docs/advanced/response-directly.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+{!../../../docs_src/response_directly/tutorial002.py!}
+```
+
+## 備考
+
+`Response` を直接返す場合、バリデーションや、変換 (シリアライズ) や、自動ドキュメントは行われません。
+
+しかし、[Additional Responses in OpenAPI](additional-responses.md){.internal-link target=_blank}に記載されたようにドキュメントを書くこともできます。
+
+後のセクションで、カスタム `Response` を使用・宣言しながら、自動的なデータ変換やドキュメンテーションを行う方法を見ることができます。
```
Comment (komtaki) [inline docs/ja/docs/advanced/response-directly.md:63]:
```suggestion
後のセクションで、カスタム `Response` を使用・宣言しながら、自動的なデータ変換やドキュメンテーションを行う方法を説明します。
```

この方が自然だと思います。

**Review by komtaki — COMMENTED** [review]
Thank your for your work. 👍 

I commented a few small suggestions.

**Comment by Attsun1031** [comment]
@komtaki Thank you for your review ! I fixed everything, so please check again.

**Review by komtaki — APPROVED** [review]
Thank you for update 👍

**Comment by tiangolo** [comment]
Awesome, thank you @Attsun1031 ! :cake: :tada: 

And thanks for the reviews @komtaki and @tokusumi :bow: :coffee:

**Pull request merged**